lambdas/bearer-token-handler/tests/bearer-token-handler.test.ts

```diff
@@ -31,9 +35,12 @@ describe("bearer-token-handler", () => {
     };
     const result = await bearerTokenHandler.handler(event, {} as Context);
     expect(result).toStrictEqual({
-      token: 12345678,
-      tokenExpiry: "1622646000000",
-      tokenExpiryInMinutes: 144000 / 60,
+      token: mockedAccessToken,
+      tokenExpiry: (
+        firstJune2021Midnight +
+        (mockedExpiryInSeconds / 2) * 1000
```

Comment:
I suppose you could do `halfOfExpiryInSeconds = 7200` instead to be even more clearer and avoid implementation in test.

For instance, you would do `expect(20/2).toBe(20/2)` but would do `expect(20/2).toBe(10)`